src/modules/src/controller_indi.c

```diff
@@ -149,7 +155,7 @@ void controllerINDI(control_t *control, setpoint_t *setpoint,
 		}
 	}
 
-	if (RATE_DO_EXECUTE(POSITION_RATE, tick)) {
+	if (RATE_DO_EXECUTE(POSITION_RATE, tick) && !outerLoopActive) {
```

Comment:
So I'm not exactly sure about this. So you made an outerloop boolean to give people the choice to use either PID or INDI for the position control? Just trying to to think if this is actually that necessary

Reply:
The main reason why I introduced this variable is to differentiate between two main use cases I needed: 
- Attitude INDI only
- Position and Attitude INDI

I can remove position PID from attitude INDI completely and at first structure the INDI controllers (into different files) as you suggested at the bottom. Thus, there will remain only two options: 
- Attitude INDI only
- Position and Attitude INDI

which are controlled by the `outerLoopActive`. 

Later, (if needed) one can add position PID.

Reply:
Hmm yeah good point. Just leave as is for now. I don't think it is a big issue on a second thought and I actually would like to try out indi/pid with my position controller script that I just pushed to the python library.